code/src/components/ThoughtList.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable no-underscore-dangle */
+
+import React from 'react';
+import { formatDistance } from 'date-fns';
+
+export const ThoughtList = ({ loading, thoughts, onLikesIncrease }) => {
+  if (loading) {
+    return <div className="Loading">Loading</div>
+  }
+
+  const handleOnLikesIncrease = (id) => {
```

Comment:
**Very good job with the like-button. It works like a charm! =)** 
What would add something extra to your app might be to change the color on the button after a happy thought has gotten a like - but that's just a nifty extra feature and no requirements in this weeks project :) 
I saw that you button works with the hover-effect if it has no likes already but it might be an idea to keep unliked thoughts with a grey background on the button and change the ones that has likes to something else - just an idea on how to improve your app even more.